contracts/CallOptionToken.sol

```diff
@@ -101,12 +99,12 @@ contract CallOptionToken is ERC20, CloberOptionToken, ReentrancyGuard, Ownable {
         require(block.timestamp <= expiresAt, "OPTION_EXPIRED");
         require(optionAmount > 0, "INVALID_AMOUNT");
 
-        uint256 collateralAmount = Math.ceilDiv(optionAmount, _underlyingPrecisionComplement);
+        uint256 collateralAmount = optionAmount;
 
         _quoteToken.safeTransferFrom(
             msg.sender,
             address(this),
-            Math.ceilDiv(optionAmount * strikePrice, _quotePrecisionComplement * _PRICE_PRECISION)
+            Math.ceilDiv(optionAmount * strikePrice, _precisionComplement)
```

Comment:
This code looks weird. I believe it should be
```suggestion
            Math.ceilDiv(optionAmount * _precisionComplement / strikePrice)
```
Note that `strikePrice` is underlying/quote.